yaptide/routes/common_sim_routes.py

```diff
@@ -168,12 +172,21 @@ def post():
         if decoded_token != sim_id:
             return yaptide_response(message="Invalid update key", code=400)
 
-        for estimator_dict in payload_dict["estimators"]:
+        for i, estimator_dict in enumerate(payload_dict["estimators"]):
```

Comment:
Maybe you could as well adjust the order in which estimators are inserted into DB ?
To keep same order as user defined in editor tab (in case of Editor projects)